alerts.go

```diff
@@ -134,51 +134,48 @@ func formatJoinedAlert(alertSet *alertSet, colorize bool) string {
 		case *mkr.MonitorConnectivity:
 			monitorMsg = ""
 		case *mkr.MonitorHostMetric:
-			switch alert.Status {
-			case "CRITICAL":
-				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", m.Metric, alert.Value, m.Operator, m.Critical)
-			case "WARNING":
-				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", m.Metric, alert.Value, m.Operator, m.Warning)
-			default:
-				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", m.Metric, alert.Value, m.Operator, m.Critical)
+			if alert.Status == "CRITICAL" && m.Critical != nil {
+				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", m.Metric, alert.Value, m.Operator, *m.Critical)
+			} else if alert.Status == "WARNING" && m.Warning != nil {
+				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", m.Metric, alert.Value, m.Operator, *m.Warning)
+			} else {
+				monitorMsg = fmt.Sprintf("%s %.2f", m.Metric, alert.Value)
 			}
 		case *mkr.MonitorServiceMetric:
-			switch alert.Status {
-			case "CRITICAL":
-				monitorMsg = fmt.Sprintf("%s %s %.2f %s %.2f", m.Service, m.Metric, alert.Value, m.Operator, m.Critical)
-			case "WARNING":
-				monitorMsg = fmt.Sprintf("%s %s %.2f %s %.2f", m.Service, m.Metric, alert.Value, m.Operator, m.Warning)
-			default:
-				monitorMsg = fmt.Sprintf("%s %s %.2f %s %.2f", m.Service, m.Metric, alert.Value, m.Operator, m.Critical)
+			if alert.Status == "CRITICAL" && m.Critical != nil {
+				monitorMsg = fmt.Sprintf("%s %s %.2f %s %.2f", m.Service, m.Metric, alert.Value, m.Operator, *m.Critical)
+			} else if alert.Status == "WARNING" && m.Warning != nil {
+				monitorMsg = fmt.Sprintf("%s %s %.2f %s %.2f", m.Service, m.Metric, alert.Value, m.Operator, *m.Warning)
+			} else {
+				monitorMsg = fmt.Sprintf("%s %s %.2f", m.Service, m.Metric, alert.Value)
```

Comment:
[memo] When `alert.Status == "CRITICAL"` but `m.Critical == nil`, the program enter this if statement.